File: Remotify.Shared/Models/SystemMetrics.cs
namespace Remotify.Shared.Models;

public class SystemMetrics
{
    public double CpuUsage { get; init; }
    public double RamUsage { get; init; }
    public double RamUsedGb { get; init; }
    public double RamTotalGb { get; init; }
    public long NetworkUploadBps { get; init; }
    public long NetworkDownloadBps { get; init; }
    public long UptimeSeconds { get; init; }
}

File: Remotify_Mobile/Remotify/Models/SystemMetrics.swift
import Foundation

/// Снимок состояния ПК из `GET /api/metrics`.
struct SystemMetrics: Decodable, Equatable, Sendable {
    /// Проценты, 0–100.
    let cpuUsage: Double
    /// Проценты, 0–100.
    let ramUsage: Double
    let ramUsedGb: Double
    let ramTotalGb: Double
    /// Байт в секунду.
    let networkUploadBps: Int64
    /// Байт в секунду.
    let networkDownloadBps: Int64
    let uptimeSeconds: Int64
}
